code/pizzabot.js

```diff
@@ -1,7 +1,266 @@
-const vegetarian = "Vegetarian Pizza"
-const hawaiian = "Hawaiian Pizza"
-const pepperoni = "Pepperoni Pizza"
+const vegetarian = "vegetarian";
+const hawaiian = "hawaiian";
+const pepperoni = "pepperoni";
+const capricciosa = "capricciosa";
+const margherita = "margherita";
+const pizzaPrice = 80;
 
-const pizzaPrice = 80
+/* FUNCTIONS */
 
-//Put your Javscript code here:
+/* Calculate the cost of order */
+const calculateTotalCost = (orderQuantity, pizzaPrice) => {
+  const orderTotal = orderQuantity * pizzaPrice;
+  return orderTotal;
+};
+
+/* Calculate cooking time */
+const calculateCookingTime = (orderQuantity) => {
```

Comment:
There is an arrow function convention which says that if we pass to our function only one argument, we can omit parentheses `()`, and rather write :

`const calculateCookingTime = orderQuantity => { ... }`

than

`const calculateCookingTime = (orderQuantity) => { ... }`

Important thing to notice - this convention should be applied ONLY when we pass one argument. If we would do this with multiple arguments, it would cause `SyntaxError` to occur